managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+
+1.  Browse to the menu: **Automation** > **Embedded Workflows** > **Credentials**.
+
+2.  Click **Configuration**, then ![Add New Credential](../images/1862.png) (**Add New Credential**).
+
+3.  Provide a **Name** for the credential.
+
+4.  Select the **Credential Type**. More fields appear depending on the type chosen.
+
+5.  Click **Add**.
+
+#### Credential Types
+
+Each credential type that is used by {{ site.data.product.title_short }} for embedded workflows is detailed in the following sections.
+
+##### Machine
+
+Machine credentials enable {{ site.data.product.title_short }} to invoke workflow steps on hosts under your management. You can specify the SSH username, and optionally provide a password, or an SSH key, or a key password. They define SSH and user-level privilege escalation access, and are used when running workflow steps on a remote host.
+
+- **Username**: The username to be used for SSH authentication.
+
+- **Password**: The actual password to be used for SSH authentication.
+
+- **SSH Private Key**: Copy or drag-and-drop the SSH private key for the machine credential.
+
+- **Private Key Passphrase**: If the SSH Private Key used is protected by a password, you can configure a Key Password for the private key.
+
+- **Privilege Escalation**: Specifies the type of escalation privilege to assign to specific users. Options include **sudo**, **su**, **pbrun**, **pfexec**.
+
+- **Privilege Escalation Username**: Enter the username to use with escalation privileges on the remote system.
+
+- **Privilege Escalation Password**: Enter the actual password to be used to authenticate the user through the selected privilege escalation type on the remote system.
+
+##### Network
+
+Network credentials are used for networking modules to connect to and manage networking devices. Network credentials have several attributes that can be configured:
+
+- **Username**: The username to use in conjunction with the network device.
+
+- **Password**: The password to use in conjunction with the network device.
+
+- **Authorize**: Select this from the Options field to add an Authorize password that signs the RSA key with a password.
+
+- **Authorize password**: If **Authorize** is checked, enter a password in the **Authorize Password** field.
+
+- **SSH Key**: Copy or drag-and-drop the actual SSH Private Key to be used to authenticate the user to the network by using SSH.
+
+- **Private key passphrase**: The actual passphrase for the private key to be used to authenticate the user to the network by using SSH.
+
+##### SCM
+
+SCM (source control) credentials are used with Projects to clone and update local source code repositories from a remote revision control system such as Git, Subversion, or Mercurial. Source Control credentials have several attributes that can be configured:
```

Comment:
@Fryguy do we actually support svn/mercurial?  I think we only have `GitRepository` at least in the modeling

Reply:
I have updated this to be: SCM (source control Management) credentials are used with Projects to clone and update local source code Git repositories.